core/src/test/scala/integration/kafka/api/PlaintextConsumerTest.scala

```diff
@@ -808,16 +808,55 @@ class PlaintextConsumerTest extends BaseConsumerTest {
 
   @ParameterizedTest(name = TestInfoUtils.TestWithParameterizedQuorumAndGroupProtocolNames)
   @MethodSource(Array("getTestQuorumAndGroupProtocolParametersAll"))
-  def testSubscribeAndCommitSync(quorum: String, groupProtocol: String): Unit = {
```

Comment:
Is this one being removed intentionally? the suggestion was only to move it to the `PlainTextConsumerCommit` file, where all tests related to committing offsets are now. Ok for me if you think it's not worth keeping, but just to make sure it's intentional.

Reply:
This appears in the PR because i didn't rebase correctly.  You've actually moved the test to here: https://github.com/apache/kafka/blob/21479a31bdff0e15cfe7ee0a4e509232ed064b41/core/src/test/scala/integration/kafka/api/PlaintextConsumerCommitTest.scala#L261